### PR TITLE
SDP-1349 adding circle sub-accounts documentation

### PR DESCRIPTION
**Pull request by reecexlm**
https://stellarorg.atlassian.net/browse/SDP-1349
Updating Circle Configuration docs to help users understand how to configure circle accounts for the SDP using sub-accounts for tenants. This will enable an organization who has a single circle account, to leverage and control funds for each agency using sub-accounts.

### CODE REVIEW

File: platforms/stellar-disbursement-platform/admin-guide/user-interface/70-circle-configutration.mdx [hunk 1]
```diff
@@ -1,25 +1,66 @@
 ---
 title: Circle Configuration
-id: circle-configutration
+id: circle-configuration
 sidebar_position: 70
 ---
 
+## Using a Shared Circle Account Across All Tenants
```
Comment (marwen-abid) [inline platforms/stellar-disbursement-platform/admin-guide/user-interface/70-circle-configutration.mdx:7]: Just to clarify, every tenant has to configure their own Circle account. You can have all tenants configuring and using the same account, but that configuration still has to be repeated for every tenant and there is no way to share it across tenants automatically .

File: platforms/stellar-disbursement-platform/admin-guide/user-interface/70-circle-configutration.mdx [hunk 2]
```diff
@@ -1,25 +1,66 @@
...
+
+### Steps to Configure Circle Sub-Accounts for Tenants
+
+#### 1. Create Circle Sub-Account
+
+Create a sub-account in Circle from your main wallet. This process is done within the Circle platform and is separate from SDP configuration. Please refer to <a href="https://developers.circle.com/circle-mint/create-a-sub-account" target="_blank" rel="noopener noreferrer">Circle's documentation</a> on how to create sub-accounts.
+
+#### 2. Create Tenant for the Agency
+
+In the SDP, add a new tenant for the agency. Refer to <a href="https://developers.stellar.org/platforms/stellar-disbursement-platform/api-reference/admin" target="_blank" rel="noopener noreferrer">the documentation on creating tenants</a> for guidance. As the initial owner, you will receive an invite link via email. Use this link to log in to SDP as the new tenant owner.
```
Comment (marwen-abid) [inline platforms/stellar-disbursement-platform/admin-guide/user-interface/70-circle-configutration.mdx:39]: `https://developers.stellar.org/platforms/stellar-disbursement-platform/api-reference/create-tenant` for create tenant call.

**Review by marwen-abid — APPROVED** [review]
LGTM

File: platforms/stellar-disbursement-platform/admin-guide/user-interface/70-circle-configutration.mdx (hunk 1 above)
Comment (reecexlm) [reply platforms/stellar-disbursement-platform/admin-guide/user-interface/70-circle-configutration.mdx:7]: ah yea ill make adjustments so this is clear. thank you

**Comment by reecexlm** [comment]
circle is discontinuing sub-accounts so this documentation update is no longer needed.

**Pull request closed without merging**